src/ui/Controllers/OrganisationController.cs

```diff
@@ -19,10 +19,12 @@ namespace GovUk.Education.ManageCourses.Ui.Controllers
     public class OrganisationController : CommonAttributesControllerBase
     {
         private readonly IManageApi _manageApi;
+        private readonly IFeatureFlags featureFlags;
```

Comment:
I think this should be `_featureFlags` by c# conventions.
Also means you don't need `this.` in the constructors.

minor